unity/PingPongGameExample/PingPongGame/Assets/Scripts/PlatformIntegration.cs

```diff
@@ -162,6 +168,12 @@ public string GetMccMnc()
       }
       return mccmnc;
     }
+
```

Comment:
Duplicate definition of GetCellID?

Reply:
I think this is less confusion in a file that defines all platforms at once, it fails if you swap target device builds. Plus I believe there's a upcoming checkin that implements them.